src/engine/ruleset/integrations/apache-http/decoders/apache-access.yml

```diff
@@ -0,0 +1,80 @@
+name: decoder/apache-access/0
+
+sources:
+  - decoder/integrations/0
+
+metadata:
+  module: apache-http
+  title: Apache HTTP Server access logs decoder
+  description: Decoder for Apache HTTP Server access logs.
+  compatibility: The Apache datasets were tested with Apache 2.4.12 and 2.4.46 and are expected to work with all versions >= 2.2.31 and >= 2.4.16 (independent from operating system).
+  versions:
+    - "2.2.31"
+    - "2.4.16"
+  author:
+    name: Wazuh Inc.
+    email: info@wazuh.com
+    date: 2022-11-15
+  references:
+    - https://httpd.apache.org/docs/2.4/logs.html
+
+check:
+  - event.original: +regex_match/\S+\s-\s\S+\s\[
```

Comment:
Leave a comment with the `check` that would be used with the configuration labels. This way, if the integration model we want is applied, the change can be automatic.

```
#TODO: Once the events arrive tagged, uncomment these lines below and remove the above regex
  # - event.module: apache-http
  # - event.dataset: apache-access
  ```

Reply:
done